### PR TITLE
Update column name in table of learn-common-operators.md

### PR DESCRIPTION
**Pull request by AlexKarch**

### CODE REVIEW

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by shsagir** [comment]
#sign-off

File: data-explorer/kusto/query/tutorials/learn-common-operators.md [hunk 1]
```diff
@@ -282,7 +282,7 @@ StormEvents
 
 **Output**
 
-|EpisodeId|Source|FriendlyName|
+|EventId|Source|FriendlyName|
 |---|---|
```
Comment (Jak-MS) [inline data-explorer/kusto/query/tutorials/learn-common-operators.md:286]:
```suggestion
|---|---|---|
```

The table is broken in markdown. This edit should fix that. Is it supposed to be broken ?

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**